proxy/proxy_script.js

```diff
@@ -0,0 +1,18 @@
+// Keep track of any local settings for the addon.
+let settings = {
+  enabled: false,
+};
+
+// Required PAC function that will be called to determine
+// if a proxy should be used.
+function FindProxyForURL(url, host) {
+  if (settings.enabled) {
+    return "DIRECT 1.2.3.4:8080"; // Update to return proxy server rules (e.g. "PROXY 1.2.3.4:8080")
+  }
+  return "DIRECT 1.2.3.4:8080";
```

Comment:
So if I visit 1.2.3.4:8080 is something meant to happen? I'm a bit confused by this.

Could we use console.log in this proxy script to do something like "For domain X, using proxy at: ..." or something. Right now its a bit baffling to me what is meant to happen.